accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-ariaLabeling-rules.ts

```diff
@@ -20,8 +20,156 @@ import { ARIADefinitions } from "../../../aria/ARIADefinitions";
 import { FragmentUtil } from "../util/fragment";
 import { DOMUtil } from "../../../dom/DOMUtil";
 import { ARIAMapper } from "../../../..";
+import { DOMWalker } from "../../../dom/DOMWalker";
 
 let a11yRulesLabeling: Rule[] = [
+    {
+        /**
+         * Description: Triggers if a landmark element has the same parent-landmark, 
+         * AND the same role as another landmark, 
+         * AND is not differentiated by aria-label or aria-labelledby. 
+         * This causes it to be difficult for a keyboard user to know the difference between two landmarks
+         * Origin:  https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA13 this is not directly part of the ARIA spec so this is only in the IBM rules as a Violation
+         * 
+         * NOTE: When we have two landmarks at the root level of the document this rule will not check for that. 
+         * For example if we have <body> <main id="main1"></main> <main id="main2"><main> </body> we do not fail this rule. 
+         * Althought this might be an accessibility error anyway. See:
+         * https://stackoverflow.com/questions/34896476/can-i-use-more-than-one-main-html-tag-in-the-same-page/34906037
+         */
+        id: "landmark_name_unique",
+        context: "aria:complementary, aria:banner, aria:contentinfo, aria:main, aria:navigation, aria:region, aria:search, aria:form",
+        run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
+            // TODO do I need to fiter out bad contentinfo nodes: The footer element is not a contentinfo landmark when it is a descendant of the following HTML5 sectioning elements: https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/HTML5.html
+            const ruleContext = context["dom"].node as Element;
+            let ownerDocument = FragmentUtil.getOwnerFragment(ruleContext);
+            let formCache = RPTUtil.getCache(ruleContext.ownerDocument, "landmark_name_unique", null);
+
+            if (!formCache) {
+                // console.log("---------ENTERING FORM CACHE")
+                formCache = {
+                    navigationNodes: [],
+                    navigationNodesComputedLabels: [],
+                    navigationNodesParents: [],
+                    navigationNodesMatchFound: []
+                }
+                let navigationNodesTemp = ownerDocument.querySelectorAll('aside,[role="complementary"], footer,[role="contentinfo"], header,[role="banner"], main,[role="main"], nav,[role="navigation"], section,[role="region"],[role="search"]');
```

Comment:
`form` is a landmark, too.

```suggestion
                let navigationNodesTemp = ownerDocument.querySelectorAll('aside,[role="complementary"], footer,[role="contentinfo"], header,[role="banner"], main,[role="main"], nav,[role="navigation"], form,[role="form"], section,[role="region"],[role="search"]');
```